### PR TITLE
Chapters 1 and 2 (existing chapters) updated for the PHP 2.0.1 client

### PR DESCRIPTION
**Pull request by alexmoore**
Updating examples for how to use the rewritten PHP 2.0.1 client.

### CODE REVIEW

File: php/Ch01-CRUD/taste-of-riak.php [hunk 1]
```diff
@@ -1,63 +1,92 @@
...
+                    ->buildObject($val1)
+                    ->atLocation($location1)
+                    ->build();
+$storeCommand1->execute();
+
 
-$val3 = array('myValue' => 3);
-$obj3 = $myBucket->newObject('three', $val3);
-$obj3->store();
+$storeCommand2 = (new Command\Builder\StoreObject($riak))
```
Comment (christophermancini) [inline php/Ch01-CRUD/taste-of-riak.php]: Similar to what I said here: https://github.com/basho/basho_docs/commit/50dd8a74f25dfa9fd301ad682dc2dc0b5441c2ae#commitcomment-10875051

You can reuse the builder when you are doing the same command repeatedly.

**Comment by christophermancini** [comment]
Other than my comment regarding reusing the builders to reduce some repetition, everything looks great.

**Comment by christophermancini** [comment]
We can circle back to polishing it off later.

**Comment by alexmoore** [comment]
Yeah, Chapter 1 is mostly "Let's learn these commands!", so repetition is OK in this case.

**Pull request merged**